node-process-argv/echo.js

```diff
@@ -1 +1 @@
-console.log(process.argv[process.argv.length - 1]);
+console.log(process.argv.splice(2).join());
```

Comment:
This is also a great solution to echoing all the parameters. It just doesn't use the fancy rest argument!

Reply:
;-;